tests/x11/reboot_gnome.pm

```diff
@@ -40,12 +40,19 @@ sub run() {
 
         send_key "ret";
 
-        wait_still_screen 8, 8;    # wait 8 seconds to make authentication window disappear after successful authentication
```

Comment:
btw. wait_still_screen 8,8 is basically sleep 8, no?

There is no way it will finish faster and there is no way it will finish later